Xlib/protocol/display.py

```diff
@@ -25,6 +25,9 @@
 import errno
 import socket
 
+# Python 2/3 compatibility.
+from six import PY3, byte2int, indexbytes
```

Comment:
It's [not Landscape.io](https://landscape.io/github/benoit-pierre/python-xlib/11/messages?classes=syntax&classes=error&classes=smell&classes=style&classes=docs).